spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -19,6 +19,7 @@
 "Affero General Public License (AGPL) v. 3": AGPL-3.0-only
 "Aladdin Free Public License (AFPL)": Aladdin
 "Amazon Software License": LicenseRef-scancode-amazon-sl
+"Android Software Development Kit License": LicenseRef-scancode-android-sdk-license
```

Comment:
Could you please provide a link in the commit message to the POM file where this was found in?

Reply:
This was intentional, as the components that use this license are not open source and therefore do not host the POM.
I could however link to a deployed POM like https://maven.google.com/com/google/android/gms/play-services-base/17.5.0/play-services-base-17.5.0.pom
Would you prefer this?

Reply:
That's what @fviernau means: Add a link to the POM which contains "Android Software Development Kit License" as a declared license, not a link to the package using that POM. Adding the link is good practice for us to verify that the mentioned declared license really exists.

Reply:
Done